### PR TITLE
doc(secret_manager): add tutorial to access rdb with credentials stored in secret manager

### PR DESCRIPTION
**Pull request by luxifer**
### Your checklist for this pull request

- [x] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

This PR adds back the tutorial to access an RDB database with the credentials stored in Secret Manager.

The example repository has been updated in [this PR](https://github.com/scaleway/tutorial-sem-terraform/pull/2) to reflect the changes in the API.

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: tutorials/sem-terraform/index.mdx [hunk 1]
```diff
@@ -0,0 +1,91 @@
+---
+meta:
+  title: Creating and accessing a Database Instance using Secret Manager and Terraform
+  description: Discover how to create and access securely a Database Instance using Secret Manager and Terraform
```
Comment (RoRoJ) [inline tutorials/sem-terraform/index.mdx:4]:
```suggestion
  description: Discover how to create and securely access a Database Instance using Secret Manager and Terraform
```

File: tutorials/sem-terraform/index.mdx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+content:
+  h1: Creating and accessing a Database Instance using Secret Manager and Terraform
+  paragraph: Discover how to create and access securely a Database Instance using Secret Manager and Terraform
```
Comment (RoRoJ) [inline tutorials/sem-terraform/index.mdx:7]:
```suggestion
  paragraph: Discover how to create and securely access a Database Instance using Secret Manager and Terraform
```

File: tutorials/sem-terraform/index.mdx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+tags: secret-manager terraform secret database-instance access
+categories:
+  - security-identity
+  - secret-manager
+  - managed-databases
+dates:
+  validation: 2023-07-10
```
Comment (RoRoJ) [inline tutorials/sem-terraform/index.mdx:14]:
```suggestion
  validation: 2024-05-30
```

File: tutorials/sem-terraform/index.mdx [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+  posted: 2024-05-30
+---
+
+In this tutorial, you will use a Terraform configuration file that builds your entire infrastructure such as a [Database Instance](/managed-databases/postgresql-and-mysql/concepts/#database-instance), a [secret](/identity-and-access-management/secret-manager/concepts/#secret) and a [version](/identity-and-access-management/secret-manager/concepts/#version) in which your database credentials are stored. You will then be able to access your database credentials securely from the Scaleway console.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
```
Comment (RoRoJ) [inline tutorials/sem-terraform/index.mdx:22]:
```suggestion
```

File: tutorials/sem-terraform/index.mdx [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+ - You have an account and logged into the [Scaleway console](https://console.scaleway.com/)
```
Comment (RoRoJ) [inline tutorials/sem-terraform/index.mdx:23]:
```suggestion
 - A Scaleway account logged into the [console](https://console.scaleway.com/)
```

File: tutorials/sem-terraform/index.mdx [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+ - You have created an [API key](/identity-and-access-management/iam/how-to/create-api-keys/)
```
Comment (RoRoJ) [inline tutorials/sem-terraform/index.mdx:24]:
```suggestion
- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
 - A valid [API key](/identity-and-access-management/iam/how-to/create-api-keys/)
```

File: tutorials/sem-terraform/index.mdx [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+ - You have [configured your environment variables](/developer-tools/devtools/reference-content/environment-variables/#recommended-scaleway-configuration-file)
```
Comment (RoRoJ) [inline tutorials/sem-terraform/index.mdx:25]:
```suggestion
 - [Configured your environment variables](/developer-tools/devtools/reference-content/environment-variables/#recommended-scaleway-configuration-file)
```

File: tutorials/sem-terraform/index.mdx [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+</Message>
+
+## Clone the GitHub repository
+
+We have created a [GitHub repository](https://github.com/scaleway/tutorial-sem-terraform) that contains all the code you need to deploy your infrastructure.
```
Comment (RoRoJ) [inline tutorials/sem-terraform/index.mdx:30]:
```suggestion
The Scaleway [GitHub repository](https://github.com/scaleway/tutorial-sem-terraform) contains all the code you need to deploy your infrastructure.
```

File: tutorials/sem-terraform/index.mdx [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+
+Open a terminal and clone the repository to your local machine using the following command:
+
+        ```bash
+        git clone git@github.com:scaleway/tutorial-sem-terraform.git
+        ```
+
+### Files and directories
```
Comment (RoRoJ) [inline tutorials/sem-terraform/index.mdx:38]:
```suggestion
### Understanding the files and directories
```

**Review by nerda-codes — APPROVED** [review]

**Review by RoRoJ — APPROVED** [review]

File: tutorials/sem-terraform/index.mdx [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+---
+
+In this tutorial, you will use a Terraform configuration file that builds your entire infrastructure such as a [Database Instance](/managed-databases/postgresql-and-mysql/concepts/#database-instance), a [secret](/identity-and-access-management/secret-manager/concepts/#secret) and a [version](/identity-and-access-management/secret-manager/concepts/#version) in which your database credentials are stored. You will then be able to access your database credentials securely from the Scaleway console.
+
+<Macro id="iam-requirements" />
+
+ - A Scaleway account logged into the [console](https://console.scaleway.com/)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+ - A valid [API key](/identity-and-access-management/iam/how-to/create-api-keys/)
+ - [Configured your environment variables](/developer-tools/devtools/reference-content/environment-variables/#recommended-scaleway-configuration-file)
```
Comment (nerda-codes) [inline tutorials/sem-terraform/index.mdx:25]: Commentaire pour moi-même: le lien renvoie vers une 404

Comment (nerda-codes) [inline tutorials/sem-terraform/index.mdx:25]:
```suggestion
```

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**